site/docs/spark.md

```diff
@@ -325,6 +325,21 @@ spark.read
 
 Time travel is not yet supported by Spark's SQL syntax.
 
+### Table names and paths
+
+Paths and table names can be loaded from the Spark3 dataframe interface. How paths/tables are loaded depends on how
+the identifier is specified. When using `spark.read().format("iceberg").path(table)` or `spark.table(table)` the `table`
+variable can take a number of forms as listed below:
+
+*  `file:/path/to/table` -> loads a HadoopTable at given path
+*  ```catalog.`file:/path/to/table` ``` -> fails. Don't set a catalog for paths
+*  ```catalog.namespace.`file:/path/to/table` ``` -> fails. Namespace doesn't exist for paths
```

Comment:
I think these lines can be removed from the docs. I don't think we need to document cases that would be parsed as strange table identifiers that were never supported.

Reply:
done